nginx_proxy/config.yaml

```diff
@@ -39,3 +39,4 @@ schema:
     active: bool
     default: str
     servers: str
+  real_ip_from: str?
```

Comment:
I see that the example docs has multiple IP addresses: http://nginx.org/en/docs/http/ngx_http_realip_module.html

Can we maybe make a list, essentially revisit what I've suggested [here](https://github.com/home-assistant/addons/pull/3274#discussion_r1520945900)?

I don't think it can be optional then, but we can consider empty list as not enabled.

Reply:
Updated the PR to handle a list; I switched to use tempio to make it easier, I'll write a PR later to update this addon to use templating for most of the other operations we're currently doing in bash directly.

Reply:
Tested this changes when there are IPs specified, and when the list is empty; both are working as expected.